html-templates/simple.mustache

```diff
@@ -294,11 +294,12 @@
 			});
 		});
 	</script>
+	<title>Salesforce Code Analyzer Report</title>
```

Comment:
Fix w3 validation error:
`Element head is missing a required instance of child element 'title'.`